services/site/src/components/challengePage/Card.tsx

```diff
@@ -83,8 +83,9 @@ const Card: React.FunctionComponent<CardProps> = ({
       )}
       <div className={clsx("p-4 pt-2 bg-background-light", "transition duration-300", "group-hover:bg-grey")}>
         <CardHeading isTopChallenge={isTopChallenge}>
+          <span className={clsx("sr-only")}>{`Challenge ${challengeNumber}`}</span>
```

Comment:
This span has to go into the Link before the link text, otherwise it is not read aloud by a screen reader when quickly tabbing through. So it should say "Challenge 4: Headings" for example.